setup.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import os
+
+from setuptools import find_packages, setup
+
+
+
+# perform the install
+setup(
+    name='tape-archive',
+    version='1.0.0',
+    description='Adds support to filesystem assetstores for storing Girder files within compressed or \
```

Comment:
Use [String literal concatenation](https://docs.python.org/2.0/ref/string-catenation.html) to split this over multiple lines. Otherwise, we'll end up with whitespace in the string.